extensions/typescript-language-features/src/features/completions.ts

```diff
@@ -97,7 +97,7 @@ class MyCompletionItem extends vscode.CompletionItem {
 		}
 
 		if (tsEntry.kindModifiers) {
-			const kindModifiers = new Set(tsEntry.kindModifiers.split(/\s+/g));
```

Comment:
Can you make this split on either whitespace or the comma? I think some ts plugins use spaces for this field